trade-finance-manager-api/src/v1/controllers/update-portal-deal-status.js

```diff
@@ -4,36 +4,39 @@ const CONSTANTS = require('../../constants');
 const updatePortalDealStatus = async (deal) => {
   const {
     _id: dealId,
+    dealType,
     submissionType,
   } = deal;
 
-  let newStatus;
+  if (dealType === CONSTANTS.DEALS.DEAL_TYPE.BSS_EWCS) {
+    let newStatus;
 
-  if (submissionType === CONSTANTS.DEALS.SUBMISSION_TYPE.AIN) {
-    newStatus = CONSTANTS.DEALS.DEAL_STATUS_PORTAL.SUBMISSION_ACKNOWLEDGED;
+    if (submissionType === CONSTANTS.DEALS.SUBMISSION_TYPE.AIN) {
+      newStatus = CONSTANTS.DEALS.DEAL_STATUS_PORTAL.SUBMISSION_ACKNOWLEDGED;
 
-    await api.updatePortalDealStatus(
-      dealId,
-      newStatus,
-    );
-  }
+      await api.updatePortalDealStatus(
+        dealId,
+        newStatus,
+      );
+    }
 
-  if (submissionType === CONSTANTS.DEALS.SUBMISSION_TYPE.MIA) {
-    newStatus = CONSTANTS.DEALS.DEAL_STATUS_PORTAL.IN_PROGRESS;
+    if (submissionType === CONSTANTS.DEALS.SUBMISSION_TYPE.MIA) {
+      newStatus = CONSTANTS.DEALS.DEAL_STATUS_PORTAL.IN_PROGRESS;
 
-    await api.updatePortalDealStatus(
-      dealId,
-      newStatus,
-    );
-  }
+      await api.updatePortalDealStatus(
+        dealId,
+        newStatus,
+      );
+    }
 
-  if (submissionType === CONSTANTS.DEALS.SUBMISSION_TYPE.MIN) {
-    newStatus = CONSTANTS.DEALS.DEAL_STATUS_PORTAL.SUBMISSION_ACKNOWLEDGED;
+    if (submissionType === CONSTANTS.DEALS.SUBMISSION_TYPE.MIN) {
+      newStatus = CONSTANTS.DEALS.DEAL_STATUS_PORTAL.SUBMISSION_ACKNOWLEDGED;
```

Comment:
Lines 14 and 32 do the same thing.  Maybe they could be combined by changing the if statement to check for both submission types?

```suggestion
if ([CONSTANTS.DEALS.SUBMISSION_TYPE.AIN, CONSTANTS.DEALS.SUBMISSION_TYPE.MIN].includes(submissionType) {
      newStatus = CONSTANTS.DEALS.DEAL_STATUS_PORTAL.SUBMISSION_ACKNOWLEDGED;
      await api.updatePortalDealStatus(
        dealId,
        newStatus,
      );
    }
```

Reply:
Good shout, i've just updated 👍